docs/source/concepts/EventWorkspace.rst

```diff
@@ -210,91 +211,7 @@ Please note these should only be done as part of a Python Algorithm, otherwise t
-   this list is not significant, since various algorithms will resort by
-   time of flight or pulse time, as needed.
-
--  Also contained in the EventList is a std::set of detector ID's. This
-   tracks which detector(s) were hit by the events in
-   the list.
-
--  The histogram bins (X axis) are also stored in EventList. The Y and E
-   histogram data are not, however, as they are calculated by the MRU
-   (below).
-
-The += operator can be used to append two EventList's together. The
-lists of TofEvent's get appended, as is the list of
-detector ID's. Don't mess with the udetmap manually if
-you start appending event lists - just call
-EventWorkpspace->makeSpectraMap to generate the spectra map (map between
-spectrum # and detector IDs) by using the info in each EventList.
-
-Most Recently Used List (MRUList)
-#################################
-
-An EventWorkspace contains a list of the 100 most-recently used
-histograms, a MRUList. This MRU caches the last histogram
-data generated for fastest display.
-
-A note about workspace index / spectrum number / detector ID
-############################################################
-
-For event workspaces there is no benefit, and only a drawback to grouping detectors in hardware, therefore most of the loading algorithms for event data **match** the workspace index and spectrum number
-in the EventWorkspace. Therefore, in an EventWorkspace, the two numbers
-will be the same, and your workspace's Axis[1] is a simple 1:1 map. As
-mentioned above, the detectorID is saved in EventList, but the
-makeSpectraMap() method generates the usual SpectraDetectorMap object.
-
-Workspace2D compatibility
-#########################
-
-EventWorkspace is designed to be able to be read (but not written to)
-like a :ref:`MatrixWorkspace <MatrixWorkspace>`. By default, if an algorithm
-performs an operation and outputs a new workspace, the
-WorkspaceFactory will create a :ref:`Workspace2D` *copy*
-of your EventWorkspace's histogram representation. If you attempt to
-change an EventWorkspace's Y or E data in place, you will get an error
-message, since that is not possible.
-
-A Note about Thread Safety
-##########################
-
-Thread safety can be surprising when using an EventWorkspace:
-
-If two threads *read* a Y histogram at the same time, this *can* cause
-problems. This is because the histogramming code will try to sort the
-event list. If two threads try to sort the same event list, you can get
-segfaults.
-
-Remember that the PARALLEL\_FOR1(), PARALLEL\_FOR2() etc. macros will
-perform the check Workspace->threadSafe() on the input EventWorkspace.
-This function will return *false* (thereby disabling parallelization) if
-any of the event lists are unsorted.
-
-You can go around this by forcing the parallel loop with a plain
-PARALLEL\_FOR() macro. **Make sure you do not read from the same
-spectrum in parallel!**
-
-.. include:: WorkspaceNavigation.txt
-
+See the Event Workspace Development documentation
```

Comment:
Make this a plain link to the developer docs. Since we aren't including the inter-sphinx there isn't another way.